proto/axelarnet/v1beta1/service.proto

```diff
@@ -71,3 +72,11 @@ service MsgService {
     };
   }
 }
+
+// QueryService defines the gRPC querier service.
+service QueryService {
+  // DepositAddress queries the a deposit address by recipient
+  rpc DepositAddress(DepositAddressRequest) returns (DepositAddressResponse) {
+    option (google.api.http).get = "/axelar/axelarnet/v1beta1/deposit_address";
```

Comment:
```suggestion
    option (google.api.http).get = "/axelar/axelarnet/v1beta1/deposit_address/{recipient_chain}/{recipient_addr}";
```

Reply:
This is used for the REST interface, right?

Reply:
done